openapi.yml

```diff
@@ -77,18 +115,17 @@ components:
           $ref: "#/components/schemas/TokenAmount"
         sellToken:
           $ref: "#/components/schemas/Address"
-        sellTokenTip:
+        tip:
           $ref: "#/components/schemas/TokenAmount"
         orderType:
-          description: Is this a buy order or sell order?
-          type: string
-          enum: [buy, sell]
+          $ref: "#/components/schemas/OrderType"
         fillType:
           description: Is this a fill-or-kill order a partially fillable order?
           type: string
           enum: [fillorkill, partial]
```

Comment:
In this official [documentation](https://swagger.io/docs/specification/data-models/enums/), they also used lower case letters. I don't have a strong opinion...

Reply:
In my experience URL parameters are typically lowercase as @josojo pointed out. I also prefer them as they they currently are (`fillorkill` and `partial`)

Reply:
okok, yes, makes sense, since for GET endpoints you won't want to use lower
the underscore part? I just felt was more readable, but also not super relevant